tests/e2e/workloads/amq/test_amq_streamer_creation.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import E2ETest, workloads
+from ocs_ci.ocs.amq import AMQ
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def test_fixture_amq(request):
+
+    amq = AMQ()
+
+    def teardown():
+        amq.cleanup()
+
+    request.addfinalizer(teardown)
+    return amq
+
+
+@workloads
+class TestAMQBasics(E2ETest):
+    @pytest.mark.polarion_id("OCS-612")
+    def test_install_amq_cephfs(self, test_fixture_amq):
+        """
+        Create amq cluster and run open messages on it
+
+        """
+
+        # Deploy amq cluster
+        test_fixture_amq.setup_amq_cluster()
```

Comment:
You wanted to directly clone upstream "strimzi-kafka-operator" to use yamls but here still it will point to ocs-workloads repo yamls as you have not updated the yamls in amq.py file.